packages/algob/src/internal/deployer.ts

```diff
@@ -114,11 +115,33 @@ class DeployerBasicMode {
    * @returns {LogicSig} multi signed logic signature from assets/<file_name>.msig
    */
   async loadMultiSig (name: string, scParams: Object): Promise<LogicSig> {
-    const [tealFile, Msig] = await readMsigFromFile(name) as string; // Get tealFile name, Msig object from .mlsig
-    const lsig = await this.loadLsig(tealFile, scParams); // Load lsig from .teal (getting logic part from lsig)
-    const decodedMsig = await decodeMsigObj(Msig);
-    lsig.msig = {};
-    Object.assign(lsig.msig, decodedMsig);
+    const lsig = await getLsig(name, scParams, this.algoOp.algodClient); // get lsig from .teal (getting logic part from lsig)
+    const Msig = await readMsigFromFile(name); // Get decoded Msig object from .msig
```

Comment:
please use lowercase names for variables. This should be handled by linter. Please make sure that your linter and github hooks work.

Reply:
Ok, noted.